README.md

```diff
@@ -49,7 +49,8 @@ on: push
 jobs:
   compile-artifact:
     name: "Compile artifact zip"
-    uses: rewindio/github-action-eb-deploy/.github/workflows/compile-artifact.yml@v2.0.0
+    # Please replace all X.Y.Z values with the latest tag
+    uses: rewindio/github-action-eb-deploy/.github/workflows/compile-artifact.yml@vX.Y.Z
```

Comment:
Might be nice to support a `vX` like actions/checkout:
https://github.com/actions/checkout/tags to get small fixes for free.

Reply:
Hm! That's a good point, but we'd need to force push the tag.
Would that pose any issue?

Reply:
We're not signing tags with GPG or preventing tags to be overridden ATM.

Reply:
I'll look to add some automation on this, as we'll want to make sure the tags stay updated properly, and for now stick with what we got to not move too quickly.